src/utils/bootstrap.js

```diff
@@ -66,3 +69,10 @@ export function teardownWorkers() {
   stopParserWorker();
   stopSearchWorker();
 }
+
+function updatePrefs(state) {
+  const pendingBreakpoints = selectors.getPendingBreakpoints(state);
+  if (prefs.pendingBreakpoints !== pendingBreakpoints) {
+    prefs.pendingBreakpoints = pendingBreakpoints;
+  }
+}
```

Comment:
I see what you are trying to do here, but I disagree with the premise that this will make things easier and that we wont have to think about it. This is difficult to test and difficult to think about. We are throwing away action.type in order to do the same thing.